doc/UsersGuide/source/packagemanager.rst

```diff
@@ -18,17 +18,35 @@ project package and a certain version of a read-only package (or library), which
 and that you drag-and-drop components from.
 
 This dependency is automatically marked in your package by adding a `uses annotation
-<https://specification.modelica.org/maint/3.5/annotations.html#version-handling>`_ at the top level. For example, if you
+<https://specification.modelica.org/maint/3.6/annotations.html#version-handling>`_ at the top level. For example, if you
 drag and drop components from MSL 4.0.0 into models of your package, the ``annotation(uses(Modelica(version="4.0.0")));``
 will be added automatically to it. This information allows OpenModelica to automatically load all the libraries
 that are required to compile the models in your own package next time you (or someone else, possibly on a different
 computer) loads your package, provided they are installed in places on the computer's file system where OpenModelica
 can find them.
 
 The default place where OpenModelica looks for packages is the so-called
-`MODELICAPATH <https://specification.modelica.org/maint/3.5/packages.html#the-modelica-library-path-modelicapath>`_.
+`MODELICAPATH <https://specification.modelica.org/maint/3.6/packages.html#the-modelica-library-path-modelicapath>`_.
 You can check where it is by typing ``getModelicaPath()`` in the Interactive Environment (Tools | OpenModelica Compiler CLI in OMEdit).
```

Comment:
You can also check the current value of `MODELICAPATH` in `Tools | Options | Libraries`.

Reply:
Updated the text accordingly.